mitosheet/src/mito/types.tsx

```diff
@@ -997,7 +997,7 @@ export interface BaseAction<Type, StaticType> {
     type: Type;
     staticType: StaticType;
     // The short title for the action. Should be title case, as you want to display it.
-    shortTitle: string
+    shortTitle?: string
```

Comment:
Why is this undefined? Let's just always have one of these, ya?

Reply:
For toolbar buttons that don't have text displayed (i.e. more/less precision)

Reply:
Can you leave a comment here about that?